Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -569,7 +569,7 @@ WebPageProxy::Internals::Internals(WebPageProxy& page)
     , tryCloseTimeoutTimer(RunLoop::main(), &page, &WebPageProxy::tryCloseTimedOut)
     , updateReportedMediaCaptureStateTimer(RunLoop::main(), &page, &WebPageProxy::updateReportedMediaCaptureState)
     , webPageID(PageIdentifier::generate())
-#if HAVE(CVDISPLAYLINK)
+#if HAVE(CVDISPLAYLINK) || PLATFORM(GTK)
```

Comment:
There are so many of these that I think it would be useful to define `HAVE_DISPLAY_LINK` for Mac and GTK and use it in all these locations.

Reply:
Thanks! This is a great idea, I'll also use it instead of `PLATFORM(GTK)` for other places where we really mean `HAVE(DISPLAY_LINK)` and will make the patch to add wpe display link support simpler too.